content/_index.html

```diff
@@ -0,0 +1,66 @@
++++
+title = "FSH School"
+linkTitle = "FSH School"
+
++++
+
+{{< blocks/cover title="Welcome to FSH School" image_anchor="top" height="full" color="orange" >}}
+<div class="mx-auto">
+	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
+		Learn More <i class="fas fa-fish ml-2"></i>
+	</a>
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" target="_blank" href="http://standardhealthrecord.org/fsh-playground/">
+		Experiment <i class="fas fa-flask "></i>
+	</a>
+	<p class="lead mt-5">The place to learn and try FHIR Shorthand</p>
+	{{< blocks/link-down color="white" >}}
+</div>
+{{< /blocks/cover >}}
+
+
+{{% blocks/lead color="primary" %}}
+FHIR Shorthand (FSH) is a domain-specific language for defining the contents of FHIR Resources and Implementation Guides.
+Explore FSH School to get the knowledge you need to get started using FSH.
+{{% /blocks/lead %}}
+
+
+{{< blocks/section color="white">}}
+{{% blocks/feature icon="far fa-file-alt" title="Declarative" %}}
+FSH allows you to define FHIR profiles in a simple, declarative language.
+No more large JSON files or spreadsheets, just human-readable text files
+which compile to the FHIR artifacts you need.
+{{% /blocks/feature %}}
+
+
+{{% blocks/feature icon="fab fa-github" title="Source control ready" %}}
+Unlike other tools for profiling FHIR, FSH works seamlessly with source control tools like
+GitHub. Maintain a history of changes, track contributions, and maintain agility by using FSH with source control.
+{{% /blocks/feature %}}
+
+
+{{% blocks/feature icon="fas fa-fire" title="State of the art" %}}
+The FSH Specification evoles as FHIR does, ensuring you have access to the latest and greatest features of FHIR. 
+Take a look at the official <a href="http://build.fhir.org/ig/HL7/fhir-shorthand/reference.html" target="_blank">FSH specification</a> 
+for more information about how FSH can be used
+to profile FHIR.
+{{% /blocks/feature %}}
+{{< /blocks/section >}}
+
+{{< blocks/section >}}
```

Comment:
Should we get permission from these guys before putting their quotes on our web site?

Reply:
These quotes are taken from the "brag sheet" that @markkramerus had, so we had permission (I assume) to use those quotes there. I can ask again if you think this context is different enough to warrant it.

Reply:
I think @markkramerus handled it for the brag sheet, so I'll let him decide if he thinks we need to request permission or give a heads up on the use of these quotes here.